docs/install.md

```diff
@@ -17,3 +17,30 @@ bin\make dev
 ```bash
 bin\console i --all
 ```
+
```

Comment:
I've created `Makefile` to simplify all of those steps below. Just type `make start` in the root of our project. If you want to login type `make login`. Refer to `Makefile` included in this repo, pls.

Reply:
Btw, you can type `make` to obtain a list of all implemented commands.

Reply:
Right. Good job! That's something what I missed. It looks like the bin/make.bat might be redundant.

Reply:
> Btw, you can type `make` to obtain a list of all implemented commands.

Yes, indeed. I prefer to use Makefile for things like those because it's OS agnostic.

Reply:
Yeah, I was thinking about removing it, but it still works for Windows users. I will probably move it somewhere else and describe it better in docs.